Toolkit/ArcGISToolkit/MeasureToolbar.swift

```diff
@@ -407,27 +413,49 @@ public class MeasureToolbar: UIToolbar, AGSGeoViewTouchDelegate {
         mapView?.sketchEditor?.clearGeometry()
     }
     
+    private var linearUnits: [AGSLinearUnit] {
```

Comment:
Could you please make this and `areaUnits` lazy? They are the same every time, correct?